browser/src/map/Clipboard.js

```diff
@@ -877,8 +851,10 @@ L.Clipboard = L.Class.extend({
 	},
 
 	_asyncAttemptNavigatorClipboardWrite: async function() {
-		const command = this._unoCommandForCopyCutPaste;
-		await this._sendCommandAndWaitForCompletion(command);
```

Comment:
We definitely do need to wait for the command to complete somehow or we come across some data races... I don't mind if you do it through some other promise, but it looks like you're not going to be waiting at all after this PR?

From your PR description I suspect you believed the promise was usused here as there's no value resolved into it? That's not true: the promise is awaited to make sure the side-effects of running the uno command are completed... see also https://github.com/CollaboraOnline/online/commit/1b3c5bfd1be0ad77882452a4ddddd8e4ef5fe807 which ran into this issue and moved (and duplicated) fetches to avoid it before the refactor